recipes/mimalloc/0001-patch-for-conda.patch

```diff
@@ -0,0 +1,25 @@
+From b826a1f164224fa5b8c53957c509b87f205b6792 Mon Sep 17 00:00:00 2001
+From: elbaro <elbaro@users.noreply.github.com>
+Date: Mon, 13 Apr 2020 18:42:14 +0900
+Subject: [PATCH] patch for conda
+
+---
+ src/alloc-override.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/alloc-override.c b/src/alloc-override.c
+index c0e7bc2..60885a8 100644
+--- a/src/alloc-override.c
++++ b/src/alloc-override.c
+@@ -173,7 +173,7 @@ void* valloc(size_t size)                                   { return mi_valloc(s
+ void* pvalloc(size_t size)                                  { return mi_pvalloc(size); }
+ void* reallocarray(void* p, size_t count, size_t size)      { return mi_reallocarray(p, count, size); }
+ void* memalign(size_t alignment, size_t size)               { return mi_memalign(alignment, size); }
+-void* aligned_alloc(size_t alignment, size_t size)          { return mi_aligned_alloc(alignment, size); }
```

Comment:
Uh, isn't this breaking the use of `mimalloc` for aligned allocations?

Reply:
Probably. This is [suggested]((https://github.com/microsoft/mimalloc/issues/219#issuecomment-609880751)) by the mimalloc author, and I am not aware of any other workaround.